### PR TITLE
Support debug mode

### PR DESCRIPTION
**Pull request by HidenoriMatsubayashi**
I've changed that `FLUTTER_RELEASE` is changed in conjunction with `CMAKE_BUILD_TYPE`.

e.g.
```
$ cmake -DUSER_PROJECT_PATH=examples/flutter-x11-client -DCMAKE_BUILD_TYPE=Debug ..
$ cmake --build .
$ FLUTTER_ENGINE_SWITCHES=2 \
     FLUTTER_ENGINE_SWITCH_1="observatory-port=12345"  \
     FLUTTER_ENGINE_SWITCH_2="disable-service-auth-codes" \
     ./flutter-x11-clinet sample/build/linux/x64/debug/bundle
```

## Related issues
- https://github.com/sony/flutter-embedded-linux/issues/50

### CODE REVIEW

**Comment by HidenoriMatsubayashi** [comment]
@Mai-Matsuura Please review this PR.

File: DEBUGGING.md [hunk 1]
```diff
@@ -12,15 +21,26 @@ flutter: Observatory listening on http://127.0.0.1:40409/k8IUol2dnPI=/
 
 ### Command line
 
-```Shell:
+```Shell
 $ flutter attach --device-id=flutter-tester --debug-uri http://127.0.0.1:40409/k8IUol2dnPI=/
 ```
 
+### How to use command line options in the Dart VM
+You can use the environment variables such as `FLUTTER_ENGINE_SWITCHES`, `FLUTTER_ENGINE_SWITCH_*` to use Dart VM command line options. If you want to fix always same observatory RUI, you can use the following command.
```
Comment (Mai-Matsuura) [inline DEBUGGING.md:29]: Is `RUI` a mistake of `URI`?

**Review by Mai-Matsuura — COMMENTED** [review]
Please see my comment.

File: DEBUGGING.md (hunk 1 above)
Comment (HidenoriMatsubayashi) [reply DEBUGGING.md]: It's typo. I've fixed it.

**Comment by Mai-Matsuura** [comment]
@HidenoriMatsubayashi  I've checked the changes and it looks fine.

**Pull request merged**